### PR TITLE
Support command to get filePath through fileId

### PR DESCRIPTION
**Pull request by maobaolong**
### What changes are proposed in this pull request?

Please outline the changes and how this PR fixes the issue.

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. If you propose a new API, clarify the use case for a new API.
  2. If you fix a bug, describe the bug.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

File: shell/src/main/java/alluxio/cli/fsadmin/command/GetFilePathCommand.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    long fileId;
+    String arg = cl.getArgs()[0];
+    try {
+      fileId = Long.parseLong(arg);
+    } catch (NumberFormatException e) {
+      throw new InvalidArgumentException(arg + " is not a valid file id.");
+    }
+
+    try {
+      String path = mFsClient.getFilePath(fileId);
```
[the remark this answers is not in the document]
Comment (maobaolong) [reply shell/src/main/java/alluxio/cli/fsadmin/command/GetFilePathCommand.java]: Yeah, it is a simple path, something like `/file`

**Comment by maobaolong** [comment]
@dbw9580 I just update this PR under the discussion between us, PTAL.

File: core/transport/src/main/proto/grpc/file_system_master.proto [hunk 2]
```diff
@@ -323,7 +323,9 @@ message FileInfo {
 }
 
 message GetFilePathPResponse {
+  /* For backward compatibility, remove path in the next release version. */
   optional string path = 1;
+  optional FileInfo fileInfo = 2;
```
Comment (dbw9580) [inline core/transport/src/main/proto/grpc/file_system_master.proto:328]: We already have `FileSystemMasterClient#getStatus` for the same thing. Should not duplicate RPCs that do the same thing.

File: shell/src/main/java/alluxio/cli/fsadmin/command/GetFilePathCommand.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    long fileId;
+    String arg = cl.getArgs()[0];
+    try {
+      fileId = Long.parseLong(arg);
+    } catch (NumberFormatException e) {
+      throw new InvalidArgumentException(arg + " is not a valid file id.");
+    }
+
+    try {
+      FileInfo fileInfo = mFsClient.getFilePath(fileId);
```
Comment (dbw9580) [inline shell/src/main/java/alluxio/cli/fsadmin/command/GetFilePathCommand.java:75]: We can achieve the same thing with two steps: 

1. get the file path from file id with `client.getFilePath`; 
2. get detailed file info by calling `client.getStatus` with the path acquired from step 1.

Step 2 is the same thing as `alluxio fs stat /file`. It should be optional and should hide behind a `--stat` flag, which suggests it does an extra step to fetch the file stats as reported by `fs stat`.

**Review by dbw9580 — CHANGES_REQUESTED** [review]

File: shell/src/main/java/alluxio/cli/fsadmin/command/GetFilePathCommand.java (hunk 3 above)
Comment (dbw9580) [reply shell/src/main/java/alluxio/cli/fsadmin/command/GetFilePathCommand.java:75]: On second thought, I think we should totally make `alluxio fs stat` accept a file id, and add a new format option to output the file path.

**Comment by dbw9580** [comment]
@maobaolong this should be superseded by #15565, can we close this PR?

**Comment by maobaolong** [comment]
@dbw9580 Thanks for remind me, close this PR

**Pull request closed without merging**